docs/network/interfaces_and_networks.md

```diff
@@ -1084,4 +1093,4 @@ manually, using ethtool. For example:
 `ethtool -L <NIC> combined #num_of_queues`
 
 More information please refer to [KVM/QEMU
-MultiQueue](http://www.linux-kvm.org/page/Multiqueue).
\ No newline at end of file
+MultiQueue](http://www.linux-kvm.org/page/Multiqueue).
```

Comment:
This change does not seem to belong in this commit.

Reply:
removed